src/main/java/com/nvidia/spark/rapids/jni/GpuSliceUtils.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2025, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids.jni;
+
+import ai.rapids.cudf.*;
+
+public class GpuSliceUtils {
+    static{
+        NativeDepsLoader.loadNativeDeps();
+    }
+
+    public static ColumnVector slice(ColumnView cv, int start, int length) {
```

Comment:
better to have comments to describe its functionality and the input and output for easier maintenance in the future.

Reply:
Personally I prefer `listsSlice` to `slice` here to distinguish it from the ones in cudf.